mushr_rhc_ros/XYHVPath path
---
bool success
